pkgs/tools/backup/bupstash/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchFromGitHub, installShellFiles, rustPlatform, ronn, pkg-config, libsodium }:
+rustPlatform.buildRustPackage rec {
+  pname = "bupstash";
+  version = "0.6.4";
+
+  src = fetchFromGitHub {
+    owner = "andrewchambers";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "013k8pr4865f5rp66fjf3a8069kmd29brxv0l20z571gy2kxs5p9";
+  };
+
+  cargoSha256 = "17cdi93q71wsqqfkpz6mxcaqqhqclsbns0g1r9mni39nikw7amv1";
+
+  nativeBuildInputs = [ ronn pkg-config installShellFiles];
+  buildInputs = [ libsodium ];
+
+  postBuild = ''
+    export RUBYOPT="-KU -E utf-8:utf-8"
```

Comment:
Is this used by ronn?

Reply:
Yeah, if its not set ronn assumes a different character encoding which fails.

Reply:
did you try only setting it for ronn itself and not export it?

Reply:
ah that would be better yes.